closed/custom/modules/java.base/Copy.gmk

```diff
@@ -206,7 +206,7 @@ ifneq ($(OPENSSL_BUNDLE_LIB_PATH), )
     # OpenSSL 1.1.1 on AIX has switched to use archive library files (natural way)
     # instead of 'libcrypto.so' files.
     # See https://github.com/openssl/openssl/pull/6487.
-    LIBCRYPTO_NAMES := libcrypto.so.3 libcrypto.so.1.1 libcrypto.so.1.0.0 libcrypto.a
+    LIBCRYPTO_NAMES := libcrypto64.so.3 libcrypto.so.3 libcrypto.so.1.1 libcrypto.so.1.0.0
```

Comment:
A problem found during testing of OpenSSL 3, which was bundled for testing purposes (although it's not normally bundled). As I understand it, if libcrypto.a is bundled then it may be found by executables exec'ed from the JVM, such as curl. However curl can only use 1.0.2 atm but libcrypto.a may contain a different version (i.e. v3), and this causes curl to fail.